ToyBox/classes/MonkeyPatchin/BagOfPatches/Alignment.cs

```diff
@@ -47,8 +47,8 @@ private static class UnitAlignment_Shift_Patch {
                             return false;
                         }
                         var vector = __instance.m_Vector;
-                        var num = (float)value / 50f;
-                        var directionVector = Traverse.Create(__instance).Method("GetDirection", new object[] { direction }).GetValue<Vector2>();
+                        var num = (float)value / __instance.Radius;
```

Comment:
This is 50 at runtime (I checked in the debugger), but OwlCat does their math on this value, so maybe there are scenarios where the radius can decrease?